board/common/rootfs/usr/libexec/infix/init.d/00-probe

```diff
@@ -110,8 +124,9 @@ class QEMUSystem:
         data = {}
         if os.path.exists(QEMUSystem.VPD):
             try:
-                data = onieprom.from_tlv(open(QEMUSystem.VPD, "rb", 0))
-            except:
+                with open(QEMUSystem.VPD, "rb", 0) as f:
+                    data = onieprom.from_tlv(f)
+            except (IOError, OSError):
```

Comment:
`onieprom.from_tlv()` can raise `AssertionError` which is no longer caught by this. We need to handle that here.

Discovered this error on a system with an unprogrammed power-board VPD

Reply:
Will fix immediately!

Reply:
Fixed, added to both cases, not just the Qemu case mentioned here.